docs/modules/operation/pages/service-assurance/polling-packages.adoc

```diff
@@ -270,13 +269,13 @@ Parameter timeout: 3000 <5>
 Service is Up on 192.168.31.100 using org.opennms.netmgt.poller.monitors.IcmpMonitor: <6>
 	response-time: 407,0000 <7>
 ----
-<1> Service and package of this test
-<2> Applied service configuration from polling package for this test
-<3> Service monitor used for this test
-<4> RRD configuration for response time measurement
-<5> Retry and timeout settings for this test
-<6> Polling result for the service polled against the IP address
-<7> Response time
+<1> Service and package of this test.
+<2> Applied service configuration from polling package for this test.
+<3> Service monitor used for this test.
+<4> RRD configuration for response time measurement.
+<5> Retry and timeout settings for this test.
+<6> Polling result for the service polled against the IP address.
+<7> Response time.
 
 == Test filters on Karaf Shell
```

Comment:
@Bonrob2 This line triggered me (but also in the past ofc): https://issues.opennms.org/browse/NMS-14421 :-)

Reply:
:-) I saw your question in the chat. Let's see what happens!